manim/config/config.py

```diff
@@ -157,9 +157,18 @@ def _parse_config(config_parser, args):
 set_rich_logger(config_parser["logger"], file_writer_config["verbosity"])
 if file_writer_config["log_to_file"]:
     # IMPORTANT note about file name : The log file name will be the scene_name get from the args (contained in file_writer_config). So it can differ from the real name of the scene.
+    scene_name_suffix = "".join(file_writer_config["scene_names"])
+    scene_file_name = os.path.basename(args.file).split(".")[
+        0
+    ]  # takes filename and removes extension
```

Comment:
Suggestion: I think this comment is unnecessary. Also, removing it will make black format this line a lot more prettily.